rln-js/index.html

```diff
@@ -34,6 +34,11 @@
         <div><h3>Commitment</h3>
             <div id="commitment"></div>
         </div>
+        <button id='register-button' type='button'>Register Credentials in Contract</button>
+    </div>
+
+    <div><h2>Wallet</h2>
```

Comment:
Add the wallet to the top of the file as one needs to connect to the wallet before being able to register the creds.